components/inputs/input-time-range.js

```diff
@@ -210,38 +282,47 @@ class InputTimeRange extends SkeletonMixin(FormElementMixin(RtlMixin(LocalizeCor
 				?label-hidden="${this.labelHidden}"
 				?required="${this.required}"
 				?skeleton="${this.skeleton}">
-				<d2l-input-time
-					?novalidate="${this.noValidate}"
-					@change="${this._handleChange}"
-					class="d2l-input-time-range-start"
-					@d2l-input-time-dropdown-toggle="${this._handleDropdownToggle}"
-					?disabled="${this.disabled}"
-					?enforce-time-intervals="${this.enforceTimeIntervals}"
-					.forceInvalid="${this.invalid}"
-					id="${this._startInputId}"
-					label="${startLabel}"
-					?label-hidden="${this.childLabelsHidden}"
-					?required="${this.required}"
-					?skeleton="${this.skeleton}"
-					time-interval="${ifDefined(timeInterval)}"
-					value="${ifDefined(this.startValue)}">
-				</d2l-input-time>
-				<d2l-input-time
-					?novalidate="${this.noValidate}"
-					@change="${this._handleChange}"
-					class="d2l-input-time-range-end"
-					@d2l-input-time-dropdown-toggle="${this._handleDropdownToggle}"
-					?disabled="${this.disabled}"
-					?enforce-time-intervals="${this.enforceTimeIntervals}"
-					.forceInvalid="${this.invalid}"
-					id="${this._endInputId}"
-					label="${endLabel}"
-					?label-hidden="${this.childLabelsHidden}"
-					?required="${this.required}"
-					?skeleton="${this.skeleton}"
-					time-interval="${ifDefined(timeInterval)}"
-					value="${ifDefined(this.endValue)}">
-				</d2l-input-time>
+				<div class="${classMap(containerClassMap)}">
+					<div class="d2l-input-time-range-start-container">
+						<d2l-input-time
+							?novalidate="${this.noValidate}"
+							@change="${this._handleChange}"
+							class="d2l-input-time-range-start"
+							@d2l-input-time-dropdown-toggle="${this._handleDropdownToggle}"
+							?disabled="${this.disabled}"
+							?enforce-time-intervals="${this.enforceTimeIntervals}"
+							.forceInvalid="${this.invalid}"
+							id="${this._startInputId}"
+							label="${startLabel}"
+							?label-hidden="${this.childLabelsHidden}"
+							?required="${this.required}"
+							?skeleton="${this.skeleton}"
+							time-interval="${ifDefined(timeInterval)}"
+							value="${ifDefined(this.startValue)}">
+						</d2l-input-time>
+					</div>
+					<div class="d2l-input-time-range-end-container">
+						<div class="d2l-input-time-range-to d2l-body-small d2l-skeletize">
+							${this.localize('components.input-time-range.to')}
```

Comment:
Not sure if this is feasible, but now that this "to" thing with the wrapping is being used in 2 places within date/time would it be possible to centralize it in a component? Like a `<d2l-date-time-range-to>` or something with a slot for each side of the range.

If that makes things even more complex then ignore me, but just seems like if some of the styles and observers could be centralized that may save some code.

Reply:
That sounds like a great idea